docs/howtos/layers/image.md

```diff
@@ -85,7 +85,7 @@ napari to seamlessly browse enormous datasets that are loaded in the right way.
 For example, here we are browsing over 100GB of lattice lightsheet data stored
 in a zarr file:
 
-![image: lattice light sheet microscopy](../../images/LLSM.webm)
+![image: lattice light sheet microscopy ](../../images/LLSM.webm)
```

Comment:
```suggestion
![napari viewer with an image layer of lattice lightsheet data opened. It can be browsed using the slider at the bottom of the viewer.](../../images/LLSM.webm)
```